src/ORM/Query.php

```diff
@@ -194,6 +195,30 @@ public function select($fields = [], $overwrite = false)
         return parent::select($fields, $overwrite);
     }
 
+    /**
+     * Select all the columns associated to $table except the $excludedFields.
+     * Excluded fields should not be aliased names.
+     *
+     *
+     * @param Table|Association $table The table to use to get an array of columns
+     * @param array $excludedFields The un-aliased column names you do not want selected from $table
+     * @param bool $overwrite Whether to reset/remove previous selected fields
+     * @return Query
+     */
+    public function selectAllExcept($table, array $excludedFields, $overwrite = false)
+    {
+        if ($table instanceof Association) {
+            $table = $table->getTarget();
+        }
+        $aliasedFields = [];
+        if ($table instanceof Table) {
+            $fields = array_diff($table->getSchema()->columns(), $excludedFields);
+            $aliasedFields = $this->aliasFields($fields);
+        }
+
+        return $this->select($aliasedFields, $overwrite);
```

Comment:
I don't see tests that toggle around overwrite, do you think adding them might be helpful?